gateway/Dockerfile

```
@@ -53,9 +45,9 @@ LABEL org.opencontainers.image.url='https://grafbase.com' \
 WORKDIR /grafbase
 
 # used curl to run a health check query against the server in a docker-compose file
-RUN apk add --no-cache curl
+RUN apt update && apt install -y curl
```

Not sure how much difference it makes, but I remember in previous job we used to delete apt stuff in the same line as `apt update` to keep layer size down.

```suggestion
RUN apt update && apt install -y curl && rm -rf /var/lib/apt/lists/*
```